src/test/java/com/corundumstudio/socketio/TransportTest.java

```diff
@@ -0,0 +1,47 @@
+package com.corundumstudio.socketio;
+
+import com.corundumstudio.socketio.Transport;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.Assert;
+
+public class TransportTest {
+
+  @Rule public ExpectedException thrown = ExpectedException.none();
```

Comment:
I'd suggest breaking the line after @Rule as this caused an error for me on a PR and involved another commit/squash/push to resolve. I will log a bug for this.